demo/play-demo-for-recording.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+
+#asciinema rec -w 2
+
+exec() {
+  echo "\$ $@"|pv -qL 10 ; "$@" ;
+  }
+
+echo "   "
+echo "   "
+echo " Simplify the deployment of Spring Boot applications using Halkyon Component Operator on Kubernetes"|pv -qL 10
+echo " In this demo, we will :"|pv -qL 10
+echo "   --> Compose & link 2 microservices: client and fruits backend"|pv -qL 10
+echo "   --> Deploy a capability such as a database and link it to the backend microservice accessing it"|pv -qL 10
+echo "   --> Code locally and next push/build on Kubernetes/OpenShift"|pv -qL 10
+echo "  "|pv -qL 10
+echo " Ready? Let's begin!"|pv -qL 10
+echo "  "
+echo "  "
+sleep 5
+
+clear && sleep 1
+echo "# Log on to the cluster using the oc client"|pv -qL 10
+sleep 2
+exec oc login https://159.69.209.188:8443 --token=5tKSUvACZ8big9XT2mONlqMU5J4w6Di6RFe9wrnJiU0
+sleep 3
+
+clear && sleep 1
+echo "# Create a new project"|pv -qL 10
+sleep 1
+exec oc new-project demo
+sleep 7
+
+clear && sleep 1
+echo "# Create a directory named demo and subsequently navigate to it"|pv -qL 10
+sleep 2
+exec mkdir demo
+sleep 1
+exec cd demo
+sleep 3
+
+clear && sleep 1
+echo "# Create a pom.xml with 2 modules corresponding each to a microservice"|pv -qL 10
+sleep 2
+echo "<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>me.fruitsand</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Spring Boot - Demo</name>
+    <description>Spring Boot - Demo</description>
+    <packaging>pom</packaging>
+    <modules>
+        <module>fruit-backend-sb</module>
+        <module>fruit-client-sb</module>
+    </modules>
+</project>"
+sleep 10
+cat > pom.xml << ENDOFFILE
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>me.fruitsand</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Spring Boot - Demo</name>
+    <description>Spring Boot - Demo</description>
+    <packaging>pom</packaging>
+    <modules>
+        <module>fruit-backend-sb</module>
+        <module>fruit-client-sb</module>
+    </modules>
+</project>
+ENDOFFILE
+clear && sleep 1
+echo "# Create a client - microservice using the REST HTTP client template proposed by the scaffolding tool"|pv -qL 10
```

Comment:
`scaffolding tool` -> `scaffolding command`. Also, why is there a `-` between `client` and `microservice`?